prboom2/src/dsda/intermission_display.c

```diff
@@ -80,14 +80,14 @@ static void dsda_UpdateIntermissionTime(dsda_split_t* split) {
     if (diff >= 2100) {
       snprintf(
         delta, sizeof(delta),
-        " (%s%d:%04.2f)",
+        " (%s%d:%05.2f)",
```

Comment:
It's just this one right? Why is the other case changed?

Reply:
I just thought it would be nice to have a consistent 0 padding in both cases, like on stopwatches: +00.03, ..., +59.97, +1:00.00
If you don't think so then I can just leave this case and discard the other one.

Reply:
Yes please do so. If I decide to line then up somehow I may do that later on

Reply:
Sure, I have left just this one and discarded the other.